addon/components/flexberry-ddau-slider.js

```diff
@@ -0,0 +1,242 @@
+  which signalizes to the route, controller, or another component, that passed value should be mutated.
+
+  Usage with manual 'change' action handling:
+  templates/my-form.hbs
+  ```handlebars
+  {{flexberry-ddau-slider
+    value=model.value
+    change=(action "onModelValueChange")
+  }}
+  ```
+
+  controllers/my-form.js
+  ```javascript
+  actions: {
+    onModelValueChange(e) {
+      // Set new value to model's 'value' property.
+      this.set('model.value', e.newValue);
+
+      // Log jQuery 'change' event object triggered after checkbox input was clicked.
+      console.log(e.originalEvent);
+    }
+  }
+  ```
+
+  Usage with {{#crossLink "FlexberryDdauSliderActionsHandlerMixin"}}flexberry-ddau-checkbox-actions-handler mixin{{/crossLink}}:
+  ```handlebars
+  {{flexberry-ddau-slider
+    value=model.value
+    change=(action "onSliderChange" "model.value")
+  }}
+  ```
+
+  controllers/my-form.js
+  ```javascript
+  import Ember from 'ember';
+  import FlexberryDdauSliderActionsHandlerMixin from 'ember-flexberry-gis/mixins/flexberry-ddau-slider-actions-handler';
+
+  export default Ember.Controller.extend(FlexberryDdauSliderActionsHandlerMixin, {
+  });
+  ```
+
+  @class FlexberryDdauSliderComponent
+  @extends <a href="http://emberjs.com/api/classes/Ember.Component.html">Ember.Component</a>
+  @uses RequiredActionsMixin
+  @uses DomActionsMixin
+  @uses DynamicActionsMixin
+  @uses DynamicPropertiesMixin
+*/
+let FlexberryDdauSliderComponent = Ember.Component.extend(
+  RequiredActionsMixin,
+  DomActionsMixin,
+  DynamicActionsMixin,
+  DynamicPropertiesMixin, {
+
+    /**
+      Component's required actions names.
+      For actions enumerated in this array an assertion exceptions will be thrown,
+      if actions handlers are not defined for them.
+
+      @property _requiredActions
+      @type String[]
+      @default ['change']
+    */
+    _requiredActionNames: ['change'],
+
+    /**
+      Reference to component's template.
+    */
+    layout,
+
+    /**
+      Reference to component's CSS-classes names.
+      Must be also a component's instance property to be available from component's .hbs template.
+    */
+    flexberryClassNames,
+
+    /**
+      Component's wrapping <div> CSS-classes names.
+
+      Any other CSS-classes can be added through component's 'class' property.
+      ```handlebars
+      {{flexberry-ddau-slider class="toggle" value=model.value change=(action "onModelValueChange")}}
+      ```
+
+      @property classNames
+      @type String[]
+      @default ['flexberry-slider', 'ui']
+    */
+    classNames: [flexberryClassNames.wrapper, 'ui'],
+
+    /**
+      Components class names bindings.
+
+      @property classNameBindings
+      @type String[]
+      @default ['readonly:read-only']
+    */
+    classNameBindings: ['readonly:read-only'],
+
+    /**
+      Component's min value.
+
+      @property min
+      @type float
```

Comment:
As I know JS doesn't have such types as 'Float', 'Int', 'Double' and so on, it has only 'Number' type.
Alsos see same other properties downside.